src/Core/BasicGraphQLExecuter.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using GraphQL.Instrumentation;
+using GraphQL.Types;
+using Microsoft.Extensions.Options;
+
+namespace GraphQL.Server
+{
+    public class BasicGraphQLExecuter<TSchema> : IGraphQLExecuter<TSchema>
+        where TSchema : ISchema
+    {
+        public TSchema Schema { get; }
+
+        private readonly IDocumentExecuter _documentExecuter;
+        private readonly GraphQLOptions _options;
+
+        public BasicGraphQLExecuter(
```

Comment:
Listeners and validation rules are configured via extension methods now. Right?

Reply:
Yes